package.json

```diff
@@ -176,7 +176,7 @@
     "supertest": "^3.1.0",
     "ts-jest": "^27.1.5",
     "ts-node": "^10.8.2",
-    "typescript": "^4.9.4",
+    "typescript": "^4.9.5",
```

Comment:
Not sure if this will resolve the `check_outdated` job - looking at [today's job](https://app.circleci.com/pipelines/github/ministryofjustice/digital-prison-services/7128/workflows/9df35daa-cecd-4441-a3e1-597e5532f770/jobs/29457) it flagged `govuk-frontend` and `typescript`. Did you attempt to bump these? TypeScript 5 may require a few changes, see [another team's PR](https://github.com/ministryofjustice/hmpps-incentives-ui/pull/430)